third_party/open-policy-agent/gatekeeper/helmify/static/values.yaml

```diff
@@ -0,0 +1,48 @@
+image:
+  repository: ghcr.io/azure/eraser-manager
+  pullPolicy: Always
+  # Overrides the image tag whose default is the chart appVersion.
+  tag: ""
+
+workerImage:
+  repository: ghcr.io/azure/eraser
+  tag: ""
+
+imagePullSecrets: []
+nameOverride: ""
+
+serviceAccount:
+  # Annotations to add to the service account
+  annotations: {}
+  # The name of the service account to use.
+  # If not set, a name is generated using the name template
+  name: ""
+
+podAnnotations: {}
+
+podSecurityContext: {}
+  # fsGroup: 2000
+
+securityContext:
+  allowPrivilegeEscalation: false
+  # capabilities:
+  #   drop:
+  #   - ALL
+  # readOnlyRootFilesystem: true
+  # runAsNonRoot: true
+  # runAsUser: 1000
+
+resources:
```

Comment:
same here for all these (resources, nodeSelector, tolerations, securityContext etc), let's move these under controller-manager. we can do a follow up with eraser, collector, etc later if that works